Use implicit rules to build write-highscores so we dont have to worry
about CC/CFLAGS/LDFLAGS/etc...

Include stdlib.h to get exit() prototype

--- write-highscores.c
+++ write-highscores.c
@@ -25,6 +25,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <stdlib.h>
 
 #define BUFFER_SIZE 1024
 
--- Makefile
+++ Makefile
@@ -4,9 +4,6 @@
 
 all: write-highscores html/wheel.png
 
-write-highscores: write-highscores.c
-	gcc -s -o write-highscores write-highscores.c
-
 html/wheel.png:
 	./makehtml
 
